client/templates/admin/fonts.css

```diff
@@ -0,0 +1,97 @@
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'gotham-bold-italic';
+    src: url('GOTHAM-BOLDITALIC.eot');
+    src: url('GOTHAM-BOLDITALIC.woff') format('woff'), 
+         url('GOTHAM-BOLDITALIC.ttf') format('truetype'), 
+         url('GOTHAM-BOLDITALIC.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+
+@font-face {
+    font-family: 'gotham-medium';
+    src: url('gotham-medium-webfont.eot');
+    src: url('gotham-medium-webfont.woff') format('woff'),
+         url('gotham-medium-webfont.ttf') format('truetype'),
+         url('gotham-medium-webfont.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+
+}
+
+/*@font-face {
+    font-family: 'gotham-medium';
+    src: url('gotham-medium-webfont.eot');
+    src: url('gotham-medium-webfont.eot?#iefix') format('embedded-opentype'),
+         url('gotham-medium-webfont.woff2') format('woff2'),
+         url('gotham-medium-webfont.woff') format('woff'),
+         url('gotham-medium-webfont.ttf') format('truetype'),
+         url('gotham-medium-webfont.svg#gotham_mediumregular') format('svg');
+    font-weight: normal;
+    font-style: normal;
+
+}*/
+
+
+
+
+@font-face {
+    font-family: 'gotham-light';
+    src: url('gotham-light-webfont.eot');
+    src: url('gotham-light-webfont.eot?#iefix') format('embedded-opentype'),
+         url('gotham-light-webfont.woff2') format('woff2'),
+         url('gotham-light-webfont.woff') format('woff'),
+         url('gotham-light-webfont.ttf') format('truetype'),
+         url('gotham-light-webfont.svg#gotham_lightregular') format('svg');
+    font-weight: normal;
+    font-style: normal;
+
+}
+
+
+
+
+
+/*@font-face {
+    font-family: 'GothamMedium';
+    src: url('fonts/GOTHAM-MEDIUM.eot');
+    src: local('☺'), url('fonts/GOTHAM-MEDIUM.woff') format('woff'), url('fonts/GOTHAM-MEDIUM.ttf') format('truetype'), url('fonts/GOTHAM-MEDIUM.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GothamMediumItalic';
+    src: url('fonts/GOTHAM-MEDIUMITALIC.eot');
+    src: local('☺'), url('fonts/GOTHAM-MEDIUMITALIC.woff') format('woff'), url('fonts/GOTHAM-MEDIUMITALIC.ttf') format('truetype'), url('fonts/GOTHAM-MEDIUMITALIC.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GothamLight';
+    src: url('fonts/GOTHAM-LIGHT.eot');
+    src: local('☺'), url('fonts/GOTHAM-LIGHT.woff') format('woff'), url('fonts/GOTHAM-LIGHT.ttf') format('truetype'), url('fonts/GOTHAM-LIGHT.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GothamLightItalic';
+    src: url('fonts/GOTHAM-LIGHTITALIC.eot');
+    src: local('☺'), url('fonts/GOTHAM-LIGHTITALIC.woff') format('woff'), url('fonts/GOTHAM-LIGHTITALIC.ttf') format('truetype'), url('fonts/GOTHAM-LIGHTITALIC.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}*/
```

Comment:
No it can be removed. 

> 16. mai 2016 kl. 16.03 skrev Sebastian Sastre <notifications@github.com>:
> 
> In client/templates/admin/fonts.css <https://github.com/dagvonkr/hof3/pull/10#discussion_r63357335>:
> 
> > +
> > +@font-face {
> > +    font-family: 'GothamLight';
> > +    src: url('fonts/GOTHAM-LIGHT.eot');
> > +    src: local('☺'), url('fonts/GOTHAM-LIGHT.woff') format('woff'), url('fonts/GOTHAM-LIGHT.ttf') format('truetype'), url('fonts/GOTHAM-LIGHT.svg') format('svg');
> > +    font-weight: normal;
> > +    font-style: normal;
> > +}
> > +
> > +@font-face {
> > +    font-family: 'GothamLightItalic';
> > +    src: url('fonts/GOTHAM-LIGHTITALIC.eot');
> > +    src: local('☺'), url('fonts/GOTHAM-LIGHTITALIC.woff') format('woff'), url('fonts/GOTHAM-LIGHTITALIC.ttf') format('truetype'), url('fonts/GOTHAM-LIGHTITALIC.svg') format('svg');
> > +    font-weight: normal;
> > +    font-style: normal;
> > +}*/
> @dagvonkr <https://github.com/dagvonkr> any reason to not remove the commented code?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/dagvonkr/hof3/pull/10/files/98f9d483bafb635547f55853ca92fc8d1d9eb88c#r63357335>